consensus/byzantine_test.go

```diff
@@ -446,8 +446,8 @@ func TestByzantineConflictingProposalsWithPartition(t *testing.T) {
 	case <-done:
 	case <-tick.C:
 		for i, reactor := range reactors {
-			t.Log(fmt.Sprintf("Consensus Reactor %v", i))
-			t.Log(fmt.Sprintf("%v", reactor))
+			t.Logf(fmt.Sprintf("Consensus Reactor %v", i))
+			t.Logf(fmt.Sprintf("%v", reactor))
```

Comment:
```suggestion
			t.Logf("Consensus Reactor %v", i)
			t.Logf("%v", reactor)
```
Sprintf is no longer needed.